### ISSUE TITLE
request.cgi.http_host is undefined in internal.cfm line 118

### ISSUE THREAD

**Issue by brentalexander** [cfwheels/cfwheels#446]
I'm running CF Wheels 1.3.2 on IIS and got the following error after my session terminated. 

Variable request.cgi.http_host is undefined in internal.cfm line 118

If I reinitialize the app, my site works until the session ends again.

To fix this, I updated line 118 to the following. I haven't had any issues since.

		// Make all cache keys domain specific
		StructInsert(arguments,ListLen(StructKeyList(arguments)) + 1,cgi.http_host,true);

Hope that helps!

**Comment by perdjurner** [comment]
Are you sure this doesn't have anything to do with your other changes to this function (I saw your other issue posted)?

**Comment by brentalexander** [comment]
This issue only popped up when I tried to upgrade to 1.3.2. To fix it, 
all I did was remove the "request." from "request.cgi.http_host" and 
since then I have not had any issues.

When I was using an earlier version of wheels, I had to make the other change that I posted about otherwise I could not get the routing to work properly with web.config. After upgrading to 1.3.2 I still needed the same change to replace that entire method in internal.cfm.

Thanks,
Brent


Date: Thu, 12 Feb 2015 04:14:50 -0800
